src/test/java/com/openlattice/rehearsal/data/DataControllerLinkingTest.kt

```diff
@@ -62,7 +62,7 @@ class DataControllerLinkingTest : SetupTestData() {
         @BeforeClass
         fun init() {
             importedEntitySets.forEach {
-                importDataSet(it.value.first, it.value.second)
+                importDataSet(it.value.first, it.value.second, true)
```

Comment:
You can change the function to be:  
`fun importDataSet(flightFileName: String, dataSource: String, loadFromFile: Boolean = true)`
and then you don't need to add true everyewhere